readme.md

```diff
@@ -192,3 +192,19 @@ You can translate the game into other languages by editing the language files in
 - [Facebook](https://www.facebook.com/OpenRCT2)
 - [RCT subreddit](http://www.reddit.com/r/rct/)
 - [OpenRCT2 subreddit](http://www.reddit.com/r/openrct2/)
+
+## Similar Projects
+
+- [openage](https://github.com/SFTtech/openage)
+- [OpenLoco](https://github.com/OpenRCT2/OpenLoco)
+- [OpenRA](https://github.com/OpenRA/OpenRA)
+- [OpenTTD](https://www.openttd.org)
```

Comment:
Is it possible to stick their logos in a table?

Reply:
| OpenRA                                                                                                        |   |   |   |   |
|---------------------------------------------------------------------------------------------------------------|---|---|---|---|
| ![](https://raw.githubusercontent.com/OpenRA/OpenRA/bleed/packaging/linux/hicolor/128x128/apps/openra-ra.png) |   |   |   |   |
| Red Alert                                                                                                     |   |   |   |   |
| https://github.com/OpenRA/OpenRA                                                                              |   |   |   |   |

Like this?

Reply:
On desktop with full width they are not centered, which I think would look better.
![image](https://user-images.githubusercontent.com/9705046/36508644-8a7a9dc2-175d-11e8-999d-78c121f335cf.png)